### DISCUSSION TITLE
DeDRM Plugin 7.2.1 with Calibre 6.10

### DISCUSSION THREAD

**Discussion by evmjohns87** [apprenticeharper/DeDRM_tools#2193] [Q&A]
Anybody had success running v7.2.1 of the plugin with Calibre 6.10? I'm trying to convert a PDF to MOBI right now but it's taking forever. Just want to make sure I'm not waiting on something that's not ultimately going to work.

**Comment by ElleKayEm** [comment]
For calibre 6, use DeDRM 10.0.3: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3

(Be aware that DRM removal happens when you first add the book to calibre.  It will try again if necessary before conversion.  But unless DeDRM wasn't enabled when the book was added, DRM should already be gone.)

**Comment by Renie-Melo** [comment]
I did the installation and everything went well. However, when converting or viewing the ebook with calibre, it shows the message that it contains DRM.
[https://ibb.co/KKwGTfD](url)

**Comment by ElleKayEm** [comment]
Post a log: https://github.com/noDRM/DeDRM_tools/blob/master/FAQs.md#i-cannot-solve-my-problem-with-the-dedrm-plugin-and-now-i-need-to-post-a-log-how-do-i-do-that
